device-data/types/basal/automated.md

```diff
@@ -0,0 +1,355 @@
+<!-- auto-generated doc! most areas *not* editable -->
+
+## Basal deliveryType: `automated`
+
+
+> Jump to...(notable fields on this type)
+
+>  - [type](#type)
+>  - [deliveryType](#deliverytype)
+>  - [duration](#duration)
+>  - [expectedDuration](#expectedduration)
+>  - [rate](#rate)
+>  - [scheduleName](#schedulename)
+
+**NB:** All fields are *required* unless otherwise noted.
+
+
+> Jump to example JSON:
+
+>  - [client example](#example-client)
+>  - [ingestion example](#example-ingestion)
+>  - [storage example](#example-storage)
+
+
+### type
+
+[ingestion, storage, client] The string `basal`.
+
+	QUICK SUMMARY
+	Required:
+		jellyfish: yes
+		platform: yes
+
+<!-- start editable commentary on type -->
+
+<!-- end editable commentary on type -->
+
+* * * * *
+
+### deliveryType
+
+[ingestion, storage, client] The string `automated`.
+
+	QUICK SUMMARY
+	Required:
+		jellyfish: nonexistent
```

Comment:
Not that it really matters, but have you tried uploading through Jellyfish? I wonder if it actually validates the `deliveryType` string.